walk-in-interview/src/main/java/com/google/job/data/JobStatus.java

```diff
@@ -0,0 +1,34 @@
+package com.google.job.data;
+
+/** Enumeration that represents the status of a job post. */
+public enum JobStatus {
+    ACTIVE("ACTIVE"),
+    DELETED("DELETED"),
+    EXPIRED("EXPIRED");
+
+    private final String statusId;
+
+    JobStatus(String statusId) {
+        this.statusId = statusId;
+    }
+
+    /**
+     * Gets the id string.
+     *
+     * @return Id string.
+     */
+    public String getStatusId() {
+        return statusId;
+    }
+
+    /**
+     * Returns the job status enum matching the provided id.
+     *
+     * @param id String id matching the enum value.
```

Comment:
nit: `@param` and `@return` are suprefluous. The "Returns..." comment above provides that information already. the throws is important though. also the `@param` is incorrect. it's not matching the enum value but the enum stable id.

Reply:
Fixed